iree/compiler/InputConversion/Common/test/linalg_quantized_matmul_to_matmul.mlir

```diff
@@ -0,0 +1,132 @@
+// RUN: iree-opt -iree-linalg-quantized-matmul-to-matmul -split-input-file %s | FileCheck %s
+
+// Tests -iree-linalg-quantized-matmul-to-matmul, converting linalg.quantized_matmul
+// ops to linalg.matmul ops plus additional arithmetic to account for any
+// nonzero zero-point.
+
+func @quantized_matmul_both_zp_0_dynamic(%lhs : tensor<?x?xi8>, %rhs : tensor<?x?xi8>, %acc : tensor<?x?xi32>) -> tensor<?x?xi32> {
+    %lhs_zp = arith.constant 0 : i32
+    %rhs_zp = arith.constant 0 : i32
+    %1 = linalg.quantized_matmul ins(%lhs, %rhs, %lhs_zp, %rhs_zp : tensor<?x?xi8>, tensor<?x?xi8>, i32, i32) outs(%acc : tensor<?x?xi32>) -> tensor<?x?xi32>
+    return %1 : tensor<?x?xi32>
+}
+// CHECK-LABEL: func @quantized_matmul_both_zp_0_dynamic
+// CHECK-SAME:    %[[LHS:.+]]: tensor<?x?xi8>, %[[RHS:.+]]: tensor<?x?xi8>
+// CHECK-SAME:    %[[ACC:.+]]: tensor<?x?xi32>
+// CHECK:       %[[MATMUL:.+]] = linalg.matmul ins(%[[LHS]], %[[RHS]] : tensor<?x?xi8>, tensor<?x?xi8>) outs(%[[ACC]] : tensor<?x?xi32>)
+// CHECK:       return %[[MATMUL]]
+// -----
+
+func @quantized_matmul_lhs_zp_0_dynamic(%lhs : tensor<?x?xi8>, %rhs : tensor<?x?xi8>, %rhs_zp : i32, %acc : tensor<?x?xi32>) -> tensor<?x?xi32> {
+    %lhs_zp = arith.constant 0 : i32
+    %1 = linalg.quantized_matmul ins(%lhs, %rhs, %lhs_zp, %rhs_zp : tensor<?x?xi8>, tensor<?x?xi8>, i32, i32) outs(%acc : tensor<?x?xi32>) -> tensor<?x?xi32>
+    return %1 : tensor<?x?xi32>
+}
+// CHECK-LABEL: func @quantized_matmul_lhs_zp_0_dynamic
+// CHECK-SAME:    %[[LHS:.+]]: tensor<?x?xi8>, %[[RHS:.+]]: tensor<?x?xi8>
+// CHECK-SAME:    %[[RHS_ZP:.+]]: i32
+// CHECK-SAME:    %[[ACC:.+]]: tensor<?x?xi32>
+// CHECK:       %[[C0_I32:.+]] = arith.constant 0 : i32
+// CHECK:       %[[MATMUL:.+]] = linalg.matmul ins(%[[LHS]], %[[RHS]] : tensor<?x?xi8>, tensor<?x?xi8>) outs(%[[ACC]] : tensor<?x?xi32>)
+// CHECK:       %[[INIT_RESULT:.+]] = linalg.init_tensor
```

Comment:
This capture is never used?

Reply:
thanks for catching that, added the use of it.